core/modules/user/user.admin.inc

```diff
@@ -399,14 +399,41 @@ function user_login_settings($form, &$form_state) {
     ),
   );
 
+  $form['user_login_page_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Login page settings'),
+  );
+  $form['user_login_page_settings']['user_login_page_tabs'] = array(
+    '#title' => t('Use tabs'),
+    '#type' => 'checkbox',
+    '#description' => t('If selected shows local tabs for "Create new account", "Log in", and "Reset passord" links. A simplified page with links is shown if unchecked.'),
```

Comment:
Typo:
```suggestion
    '#description' => t('If selected shows local tabs for "Create new account", "Log in", and "Reset password" links. A simplified page with links is shown if unchecked.'),
```

Reply:
...however, if this was converted to a `Login links style` radio set with two options, this lengthy help text would either not be required, or be repurposed to explain details/differences about the two options. Something like this:
- Horizontal tabs
  > Typically placed over the login form.
- Links
  > Typically placed below the login form.

Reply:
Could do